package.json

```diff
@@ -24,6 +24,7 @@
     "@angular/platform-browser": "^7.2.15",
     "@angular/platform-browser-dynamic": "^7.2.15",
     "@angular/router": "^7.2.15",
+    "angular-froala-wysiwyg": "^3.0.0-rc.2-1",
```

Comment:
It does not seem to allow ordered and unordered lists.
I think It's a useful feature to have and might be needed sometimes. Like here, https://evalai.cloudcv.org/web/challenges/challenge-page/392/overview

Reply:
yes, we can change the editor later on.